README.md

```diff
@@ -19,17 +19,17 @@ following section.
  * `crypto_policies_policy`
 
 Use this variable to specify the desired crypto policy on the target system,
-which can be either the base policy or a base policy with policy submodule(s)
+which can be either the base policy or a base policy with subpolicies
 as accepted by the `update-crypto-policies` tool. For example `FUTURE` or
-`DEFAULT:NO-SHA1:GOST`. The specified base policy and policy submodule(s)
+`DEFAULT:NO-SHA1:GOST`. The specified base policy and subpolicies
 must be available on the target system.
 
 The default value is `null` meaning the configuration is not changed and
 the role will just collect the facts below.
 
 The list of available base policies on the target system can be found in the
-`crypto_policies_available_policies` variable and the list of available policy
-submodules can be found in the `crypto_policies_available_modules` variable.
+`crypto_policies_available_policies` variable and the list of available
+subpolicies can be found in the `crypto_policies_available_subpolicies` variable.
```

Comment:
Do we want to mention `crypto_policies_available_modules` is deprecated and should be replaced with `crypto_policies_available_subpolicies` in README since it was included in the rhel-8.4 package?

Reply:
I think I did that several paragraphs below.

Reply:
> I think I did that several paragraphs below.

You are right. Thanks for the confirmation.